### ISSUE TITLE
Empty GitHub repo gives an error

### ISSUE THREAD

**Issue by hartlomiej** [ChappIO/git-backup#27]
To reproduce, star https://github.com/hartlomiej/testtesttest and run git-backup
```
2024/10/12 01:24:04 Discovered hartlomiej/testtesttest
2024/10/12 01:24:04 Failed to clone: reference not found
```
Avoidable via `-backup.fail-at-end`, but Ideally it should log such information without quitting or create an empty dir

**Comment by iamhere2** [comment]
@hartlomiej Could you please elaborate how to avoid this error? Bumped into the same issue.
I run git-backup with docker using command like 
`docker run -v {local-path}:/backups ghcr.io/chappio/git-backup:1`

**Comment by ChappIO** [comment]
@iamhere2 The workaround @hartlomiej mentioned was using the `-backup.fail-at-end` option. This option will continue making backups if an error occurs and throw all errors at the end. Meaning only the failed repository will have been skipped:

`docker run -v {local-path}:/backups ghcr.io/chappio/git-backup:1 -backup.fail-at-end`

However, I just released version v1.6.1 which should fix this issue. So if you run `docker pull ghcr.io/chappio/git-backup:1` you should receive the updated docker image.

**Comment by hartlomiej** [comment]
@ChappIO Works great now! I can see it note empty repos and skip them. Thanks a lot.
